package.json

```diff
@@ -54,5 +54,8 @@
     "desktop": "assets/screenshot-desktop.jpg",
     "mobile": "assets/screenshot-mobile.jpg"
   },
-  "version": "0.9.0"
+  "version": "0.9.0",
+  "devDependencies": {
```

Comment:
This `package.json` has metadata only relevant for Ghost, please move this dependency to the `package.json` file located in `src`.

Reply:
Sure! Will update this tonight, and rebase :)